semantickernel-api-builders/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This does proliferate the number of maven modules we publish to create a single class module, is there some dependency restriction that requires this module?